### PR TITLE
Id mapping of VCI to trust framwork.

### PR DESCRIPTION
**Pull request by cjbuchanan**
This will complete issue #59 when accepted.

### CODE REVIEW

**Comment by agropper** [comment]
Thank you @cjbuchanan for this analysis of @jmandel's protocol choices [wiki](https://github.com/smart-on-fhir/health-cards/wiki/Interop-Stack#the-health-cards-spec-makes-the-following-choices).

Please consider using this analytical approach for an alternate standards and privacy design proposal:

Whereas:

**Goal:** Speed and ease of adoption of W3C VCs for COVID vaccination
**Stipulate:** “The justifiable parties are limited to patients, issuers, and validator HCPs utilizing VCI standards to ingest HL7 FHIR v4 data”
**Document:** VCI standards and trust-related issues to meet the Goal by facilitating “very-near-term-adoption” issues.
**Consider:** Various DIDs may be useful for some of the justifiable parties and their relationships but are a nice-to-have and not essential to the Goal.
**Consider:** That VCs, FHIR, JWS, JWE, and OAuth2, and arguably DID Core are the only ‘real’ standards in this use-case. Protocols and methods such as DIDcomm, DID-SIOP, did:ion, did:web, did:key, and GNAP are works-in-progress but all are under intense development and evolution at this time.

Proposed:

**Health Cards would be more likely to succeed at rapidly introducing SSI principles to COVID vaccinations if the core component associated with a patient was a fiduciary GNAP authorization server as a service.** See also the thread https://github.com/smart-on-fhir/health-cards/pull/49

**Comment by cjbuchanan** [comment]
@agropper -- good recommendation.  I think that makes it clearer for everyone.

File: docs/VCI-trust-framework-mapping.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+# Health Cards to Trust Framework Model Mapping
+
+This document is written in language that should be clear to identity providers participating in the Decentralized Identity ecosystem.
+
+## Health Cards
+
+While Health-Cards utilizes [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) and [Decentralized Identifiers](https://w3c.github.io/did-core/), it is not a decentralized identity system. The overall approach is to wrap FHIR 4.0 data in a Verifiable Credential (VC) format [(JWS)](https://tools.ietf.org/html/rfc7515) which is signed by the issuer of the VC. This data will be ingestible by any Health-Cards compatible Healthcare Provider (HCP). The Verifiable Credential does not authenticate the holder utilizing the VC, but instead, patient matching is done with the FHIR 4.0 API compatible patient record wrapped in the VC. This method allows data transport and control by the patient with minimal impact on existing systems and processes.
```
Comment (Sakurann) [inline docs/VCI-trust-framework-mapping.md:7]: Health-Cards do not use DIDs AFAIK

Comment (jmandel) [reply docs/VCI-trust-framework-mapping.md:7]: Work on the trust framework mapping document stalled in early 2021; @cjbuchanan should we remove this document from the repo?

**Comment by agropper** [comment]
Per my Feb 6 https://github.com/smart-on-fhir/health-cards/pull/63#issuecomment-774457639, let's consider the scope of FHIR 4.0 compatibility we are proposing:
- MUST a Health Card always fit the FHIR 4.0 data by value or MAY it be by reference to the issuer?
- MUST a Health Card presume patient matching or MAY it include a biometric? 
- How important is it for Health Cards to support delegated access to the FHIR 4.0 data (see the [Cruise Ship Use Case](https://docs.google.com/document/d/1Mt1m-fVaSxY6QC3mgHcrNoNH-EnOTRepcwYd9hyjmxM/edit) )

File: docs/VCI-trust-framework-mapping.md (hunk 1 above)
Comment (jmandel) [reply docs/VCI-trust-framework-mapping.md:7]: @cjbuchanan I think this is at your discretion; this isn't something our implementer community has been clamoring for, but I'd be happy to include it if you think it'd be valuable.

**Review by woodynj — APPROVED** [review]

**Pull request closed without merging**